editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -4945,198 +4945,198 @@ VisualShaderEditor::VisualShaderEditor() {
-	const String input_param_for_vertex_shader_mode = TTR("'%s' input parameter for vertex shader mode.");
-	const String input_param_for_start_shader_mode = TTR("'%s' input parameter for start shader mode.");
-	const String input_param_for_process_shader_mode = TTR("'%s' input parameter for process shader mode.");
-	const String input_param_for_collide_shader_mode = TTR("'%s' input parameter for collide shader mode.");
-	const String input_param_for_start_and_process_shader_mode = TTR("'%s' input parameter for start and process shader modes.");
-	const String input_param_for_process_and_collide_shader_mode = TTR("'%s' input parameter for process and collide shader modes.");
-	const String input_param_for_vertex_and_fragment_shader_mode = TTR("'%s' input parameter for vertex and fragment shader modes.");
+	const String input_param_for_vertex_and_fragment_shader_modes = TTR("'%s' input parameter for vertex and fragment shader modes.\n\nTranslated to '%s' in Godot Shading Language.");
+	const String input_param_for_fragment_and_light_shader_modes = TTR("'%s' input parameter for fragment and light shader modes.\n\nTranslated to '%s' in Godot Shading Language.");
+	const String input_param_for_fragment_shader_mode = TTR("'%s' input parameter for fragment shader mode.\n\nTranslated to '%s' in Godot Shading Language.");
+	const String input_param_for_sky_shader_mode = TTR("'%s' input parameter for sky shader mode.\n\nTranslated to '%s' in Godot Shading Language.");
+	const String input_param_for_fog_shader_mode = TTR("'%s' input parameter for fog shader mode.\n\nTranslated to '%s' in Godot Shading Language.");
+	const String input_param_for_light_shader_mode = TTR("'%s' input parameter for light shader mode.\n\nTranslated to '%s' in Godot Shading Language.");
+	const String input_param_for_vertex_shader_mode = TTR("'%s' input parameter for vertex shader mode.\n\nTranslated to '%s' in Godot Shading Language.");
+	const String input_param_for_start_shader_mode = TTR("'%s' input parameter for start shader mode.\n\nTranslated to '%s' in Godot Shading Language.");
+	const String input_param_for_process_shader_mode = TTR("'%s' input parameter for process shader mode.\n\nTranslated to '%s' in Godot Shading Language.");
+	const String input_param_for_collide_shader_mode = TTR("'%s' input parameter for collide shader mode.\n\nTranslated to '%s' in Godot Shading Language.");
+	const String input_param_for_start_and_process_shader_mode = TTR("'%s' input parameter for start and process shader modes.\n\nTranslated to '%s' in Godot Shading Language.");
+	const String input_param_for_process_and_collide_shader_mode = TTR("'%s' input parameter for process and collide shader modes.\n\nTranslated to '%s' in Godot Shading Language.");
+	const String input_param_for_vertex_and_fragment_shader_mode = TTR("'%s' input parameter for vertex and fragment shader modes.\n\nTranslated to '%s' in Godot Shading Language.");
```

Comment:
Maybe you can factor out `Translated to '%s' in Godot Shading Language.` into its own `TTR()` so that translators only need to translate it once.
So something like:
```cpp
const String translation_gdsl = "\n\n" + TTR("Translated to '%s' in Godot Shading Language.");
const String input_param_for_vertex_and_fragment_shader_modes = TTR("'%s' input parameter for vertex and fragment shader modes.") + translation_gdsl;
...
```

Reply:
Done